geoalchemy2/elements.py

```diff
@@ -297,9 +307,9 @@ class RasterElement(_SpatialElement):
     most cases you won't need to create ``RasterElement`` instances yourself.
     """
 
-    geom_from_extended_version = "raster"
+    geom_from_extended_version: str = "raster"
 
-    def __init__(self, data):
+    def __init__(self, data) -> None:
```

Comment:
What type should we use to annotate `data` here?

Reply:
I think it should be `str` and `memoryview`

Reply:
See 2f7b1ad48df96c51ef7e59dc834918b0e29ad668